### PR TITLE
fix:Responsiveness testimonials details

### LINKED ISSUE #828: [bug]:Testimonials content is out of div
### Describe the bug

Hello @AnuragThePathak ,
Testimonials content is out of div  for all small devices. check it asap 
Link:https://wemakedevs.org/courses/dsa
I want to work on this issue... assign me this issue

### To Reproduce

_No response_

### Expected Behavior

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by techmannih**
## Fixes Issue #828 

closes #828 

## Changes proposed
Now, Testimonials section is fully responsive with above of 320px , check my PR and also merged it

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes
Screenshot 2023-11-27 114054
Screenshot 2023-11-27 114112

## Summary by CodeRabbit

- **Style**
	- Improved responsive design for `TestimonialCard` component, ensuring better layout on medium-sized screens.
	- Adjusted width constraints for testimonials to enhance visual presentation across various devices.

### CODE REVIEW

File: src/components/Cards/TestimonialCard.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   return (
-    <div className='mb-10 flex h-full w-full flex-col items-center justify-center rounded-lg border border-content/20 bg-gradient-to-br from-content/0 to-content/10 p-4 py-8 text-center lg:flex-1'>
+    <div className='mb-10 flex h-full w-full max:md:w-72 flex-col items-center justify-center rounded-lg border border-content/20 bg-gradient-to-br from-content/0 to-content/10 p-4 py-8 text-center lg:flex-1'>
       <img
         alt='testimonial'
         className=' mb-2 inline-block h-20 w-20 rounded-full border-2 border-base-100 bg-base-100/10 object-cover object-center'
         src={src}
       />
-      <p className='text-lg'>{description}</p>
+      <p className='text-lg max-md:w-64'>{description}</p>
```
[the remark this answers is not in the document]
Comment (AnuragThePathak) [reply src/components/Cards/TestimonialCard.jsx:9]: @techmannih you need to fix this maybe? The max-md class doesn't exist in my knowledge. Also make sure that the UI looks appropriate after you correctly set the class.

File: src/components/layout/Testimonials.jsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ const Testimonials = ({
         {tweetId ? (
           <TweetCarousael tweetId={tweetId} setTweetId={setTweetId} />
         ) : (
-          <div className='flex flex-col gap-4'>
+          <div className='flex flex-col gap-4 max-md:w-80'>
```
Comment (AnuragThePathak) [inline src/components/layout/Testimonials.jsx:35]: Change this one as well.

File: src/components/Cards/TestimonialCard.jsx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 const TestimonialCard = ({ src, description, from }) => {
   return (
-    <div className='mb-10 flex h-full w-full flex-col items-center justify-center rounded-lg border border-content/20 bg-gradient-to-br from-content/0 to-content/10 p-4 py-8 text-center lg:flex-1'>
+    <div className='mb-10 flex h-full w-full max:md:w-72 flex-col items-center justify-center rounded-lg border border-content/20 bg-gradient-to-br from-content/0 to-content/10 p-4 py-8 text-center lg:flex-1'>
```
Comment (AnuragThePathak) [inline src/components/Cards/TestimonialCard.jsx:3]: And this one too.

File: src/components/layout/Testimonials.jsx (hunk 2 above)
Comment (AnuragThePathak) [inline src/components/layout/Testimonials.jsx:35]: Correct it.

**Review by AnuragThePathak — CHANGES_REQUESTED** [review]

File: src/components/layout/Testimonials.jsx (hunk 2 above)
Comment (techmannih) [reply src/components/layout/Testimonials.jsx:35]: when I change this class, code is working locally

**Comment by inclinedadarsh** [comment]
Heyy @techmannih 
I'll be closing the PR for now, as we are many more commits ahead.
Also, the testimonial section seems pretty much responsive to me already.

Feel free to comment and correct me if I'm wrong. We'll make sure to consider that.
However, I'll have to close the PR for now.

Thank you for your contribution!

**Pull request closed without merging**